Fixes

Dylanben144 opened the pull request:
Fixes a basic UI problem and page view cron job

Dylanben256 committed:
Acquired button showing for guest fix

Dylanben256 committed:
Fix page view tracking cron job

D-Simona-G commented on ckanext/privatedatasets/plugin.py:
```diff
@@ -341,18 +341,22 @@ def before_view(self, pkg_dict):
+                    pkg_dict['resources'].remove(resource)
+                    pkg_dict = self.before_view(pkg_dict)
+            except:
+               print ("problem fixed")
```
Do we need this print here?

Dylanben144 replied:
We need something to fill the except and state the problem has occurred (Will help with debugging future occurrence of this problem). I will change to a better print message

Dylanben144 committed:
Small change based on pull request comments

Better print message to display location of problem. Could help with future debugging of out of context problems.

D-Simona-G commented on ckanext/privatedatasets/plugin.py:
```diff
@@ -386,18 +390,22 @@ def before_delete(self, context, resource, resources):
+            except tk.NotAuthorized:
+                resource_dict.clear()
+        except:
+            print ("problem fixed")
```
Same thing here, sorry I haven't noticed it the first time around

Dylanben144 replied:
Added same fix

Dylanben144 committed:
Small change based on pull request comments

Added better print to better describe problem
